src/pages/Homepage.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #97d197;
+}
+
```

Comment:
- [ ] As requested by the previous reviewer,  you should follow the style guideline which should include the font-family:

<img width="722" alt="image" src="https://user-images.githubusercontent.com/77190203/175142060-cd732a24-846b-470a-af7a-80c0afade65c.png">